owner/manager_test.go

```diff
@@ -194,7 +190,7 @@ func TestCluster(t *testing.T) {
 	}
 	err = d.Stop()
 	if err != nil {
```

Comment:
You can refactor the code as

```go
require.NoError(t, err)
```

ditto others.

Reply:
Thank you, fixed.